CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.11.0
+
+- Addded support for cluster domain config option. Defaults to svc.cluster.local. [#339](https://github.com/ngrok/kubernetes-ingress-controller/pull/339)
+
```

Comment:
Hi @fr6nco, thanks for making a PR for this. Will you please remove this change to release a new version? We'll take care of that as maintainers — thanks 🙏